lib/membrane_matroska/muxer.ex

```diff
@@ -34,7 +33,7 @@ defmodule Membrane.Matroska.Muxer do
     demand_unit: :buffers,
     accepted_format:
       any_of(
-        MP4.Payload,
+        %H264{stream_structure: {avc, _dcr}} when avc in [:avc1, :avc3],
```

Comment:
Shouldn't it be: 
```suggestion
        %H264{stream_structure: {avc, _dcr}, nalu_in_metadata: true} when avc in [:avc1, :avc3],
```
?